pkg/operator/controller/status/controller.go

```diff
@@ -313,8 +331,24 @@ func checkAllIngressesAvailable(ingresses []operatorv1.IngressController) bool {
 	return len(ingresses) != 0
 }
 
-// computeOperatorDegradedCondition computes the operator's current Degraded status state.
-func computeOperatorDegradedCondition(ingresses []operatorv1.IngressController) configv1.ClusterOperatorStatusCondition {
+func computeOperatorDegradedCondition(ingresses []operatorv1.IngressController, unmanagedGatewayAPICRDNames string) configv1.ClusterOperatorStatusCondition {
+	condition := computeIngressControllerDegradedCondition(ingresses)
+	if len(unmanagedGatewayAPICRDNames) > 0 {
+		gwapiCRDMsg := fmt.Sprintf("Unmanaged Gateway API CRDs found: %s.", unmanagedGatewayAPICRDNames)
+		if condition.Status != configv1.ConditionTrue {
+			condition.Status = configv1.ConditionTrue
+			condition.Reason = "GatewayAPICRDsDegraded"
+			condition.Message = gwapiCRDMsg
+		} else {
+			condition.Reason = "MultipleComponentsDegraded"
+			condition.Message = condition.Message + gwapiCRDMsg
+		}
+	}
+	return condition
+}
+
+// computeIngressControllerDegradedCondition computes the operator's current Degraded status state.
+func computeIngressControllerDegradedCondition(ingresses []operatorv1.IngressController) configv1.ClusterOperatorStatusCondition {
```

Comment:
It looks like you moved the godoc from `computeOperatorDegradedCondition` to `computeIngressControllerDegradedCondition`, but the comment still describes the former.  Please make sure that each function has an accurate godoc comment.

Reply:
Updated the godocs.